src/index.js

```diff
@@ -13,10 +14,10 @@ class TelegramLoginButton extends React.Component {
 
     const script = document.createElement('script');
     script.src = 'https://telegram.org/js/telegram-widget.js?2';
-    script.setAttribute('data-telegram-login', botName || 'samplebot');
-    script.setAttribute('data-size', dataSize || 'large');
-    script.setAttribute('data-request-access', requestAccess || 'write');
-    script.setAttribute('data-userpic', !usePic);
+    script.setAttribute('data-telegram-login', botName);
+    script.setAttribute('data-size', dataSize);
+    script.setAttribute('data-request-access', requestAccess);
+    script.setAttribute('data-userpic', !!usePic);
```

Comment:
Can we use `usePic` instead `!!usePic` here?

Reply:
Actually, yes now. Because I added default props so we won't have undefined.